src/status_im/components/drawer/view.cljs

```diff
@@ -118,8 +111,8 @@
   (fn [_ row-id _]
     (when (< row-id (dec transactions-count))
       (list-item
-        ^{:key row-id}
-        [common/separator {} st/transactions-list-separator]))))
+       ^{:key row-id}
```

Comment:
thanks yet another symbol to add to my emacs config, I need to check later if there isn't a general solution for that I noticed you like to limit the width of the lines by breaking lines after the function name :)